_data/l10n.yml

```
@@ -307,107 +307,132 @@ it:
 I range di ogni misura possono essere trovati nel file [vitality-ranges.yml](https://github.com/italia/publiccode-crawler/blob/main/vitality-ranges.yml)."
     publiccode:
       categories:
-        accounting: accounting
-        agile-project-management: agile project management
-        applicant-tracking: applicant tracking
-        application-development: application development
-        appointment-scheduling: appointment scheduling
+        accounting: contabilità
+        agile-project-management: gestione agile di progetti
+        applicant-tracking: monitoraggio candidati
+        application-development: sviluppo applicazioni
+        appointment-scheduling: programmazione appuntamenti
         backup: backup
-        billing-and-invoicing: billing and invoicing
+        billing-and-invoicing: fatturazione e scontrini
```

```suggestion
        billing-and-invoicing: Fatturazione
```

(the original description is probably redundant)